mod-infra/src/main/scala/at/forsyte/apalache/infra/ExitCodes.scala

```diff
@@ -0,0 +1,57 @@
+package at.forsyte.apalache.infra
+
+import tlc2.output.EC
+
+/**
+ * Exit codes that are reported by Tool.
```

Comment:
Should `Tool` be linked or else maybe write "by the CLI tool" or something? Depending on whether this is referring to the Scala object or the program entrypoint more generally?

Reply:
The problem I have here, is to properly `[[link]]` in Scaladoc, the name must be visible/imported. But then the compiler complains about the unused import.

I actually hit this a couple of times now, any ideas how to resolve this?
I'd rather not silence a compiler warning just to have a relatively innocent link in a comment.

Maybe this is a compromise?
```suggestion
 * Exit codes that are reported by `Tool`.
```

Reply:
Yep, that seems to make sense to me! It’d a a small thing, but I think this makes it read a bit better.